python/lib/cloudutils/syscfg.py

```diff
@@ -144,9 +146,14 @@ def __init__(self, env):
 
 class sysConfigAgentRedhat7Base(sysConfigAgent):
     def __init__(self, env):
-        self.svo = serviceOpsRedhat7()
+        self.svo = serviceOpsRedhat7Later()
```

Comment:
Perhaps refactor the name? Unless this will handle for EL8 as well.

Reply:
Yes this will handle EL8 as well, therefore, the name change